capplets/display/xrandr-capplet.c

```diff
@@ -452,6 +457,52 @@ mirror_screens_is_supported (App *app)
     return mirror_is_supported;
 }
 
+static void
+on_scale_btn_active_changed_cb (GtkWidget *widget,
+                                App       *app)
+{
+    gint scale;
+
+    if (!gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON (widget)))
+        return;
+
+    scale = g_object_get_data (G_OBJECT (widget), "scale");
+    g_settings_set_int (app->scale_settings, WINDOW_SCALE_KEY, scale);
+}
+
+static void
+rebuild_scale_window (App *app)
+{
+    GtkRadioButton *group = NULL;
+    GtkWidget      *scale_btn;
+    gint32          scale;
+    int             i;
+    const char     *button_label[] = {_("auto detect"), "100%", "200%", NULL};
+
+    gtk_container_foreach (GTK_CONTAINER (app->scale_bbox), (GtkCallback) gtk_widget_destroy, NULL);
```

Comment:
New build warning
```
xrandr-capplet.c:482:61: warning: cast between incompatible function types from 'void (*)(GtkWidget *)' {aka 'void (*)(struct _GtkWidget *)'} to 'void (*)(GtkWidget *, void *)' {aka 'void (*)(struct _GtkWidget *, void *)'} [-Wcast-function-type]
```

Reply:
The warning has been fixed

Reply:
Please confirm whether to change this problem